shell/platform/windows/key_event_handler.cc

```diff
@@ -122,7 +122,9 @@ KeyEventHandler::KeyEventHandler(flutter::BinaryMessenger* messenger,
               kChannelName,
               &flutter::JsonMessageCodec::GetInstance())),
       send_input_(send_input) {
+#ifndef WINUWP
```

Comment:
Add an explanatory comment/todo here referring to the null delegate.

Reply:
Fixed in next commit.